R/quarto.qmd

```diff
@@ -0,0 +1,5 @@
+# Plumber {.unnumbered}
+
+Quarto.org has an excellent example [here](https://quarto.org/docs/publishing/rstudio-connect.html) of how to publish a Quarto doc and publish it to Connect.
+
+_[Question] - What sort of Quarto doc makes the most sense to show how to create in a guide intended for SysAdmins?_
```

Comment:
It would be worth providing such links and noting that publishing a quarto doc will be consistent with Rmd publishing for push button workflows though.

also cover publishing from quarto to connect via `quarto publish` cli command